src/pages/groupComparison/GroupComparisonUtils.ts

```diff
@@ -2,14 +2,17 @@ import ListIndexedMap from 'shared/lib/ListIndexedMap';
 import { MobxPromise } from 'mobxpromise/dist/src/MobxPromise';
 import {SampleIdentifier, Sample, PatientIdentifier} from "../../shared/api/generated/CBioPortalAPI";
 import _ from "lodash";
+import {ResultsViewTab} from "../resultsView/ResultsViewPageHelpers";
+import {GroupComparisonTab} from "./GroupComparisonPage";
+import {StudyViewFilter} from "../../shared/api/generated/CBioPortalAPIInternal";
 
 export type SampleGroup = {
     // mandatory:
     id:string, // unique identifier
     sampleIdentifiers:SampleIdentifier[], // samples in the group
+    name:string, // display name
 
     // optional:
-    name?:string, // display name
     color?: string; // color for charts
```

Comment:
maybe slight more descrtive name for color?

Reply:
I'll do this in a new PR where I also address some refactoring to be done around colors